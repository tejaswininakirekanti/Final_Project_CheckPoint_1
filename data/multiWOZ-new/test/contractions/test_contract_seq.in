i 'd like to leave from stansted airport after 11:45 .
i 'd like to leave anytime after 20:15 .
i 'm looking for a train going to cambridge .
i 'd like to leave broxbourne on tuesday .
yes i 'd like a ticket for one .
i 'm looking for a train departing from london kings cross
i 'm travelling on monday and need to leave after 16:30 .
i 'm departing from birmingham new street and going to cambridge .
i 'd like to go on saturday .
i 'm looking for a train from cambridge on monday .
i 'm looking for a train that goes to london liverpool street and leaves after 17:15 .
i 'm looking for a train that departs from cambridge on thursday .
i 'll be leaving from cambridge .
i 'll depart on wednesday .
i 'd like to find a train that 's leaving cambridge on monday .
i need to find a train that 's going to arrive in stevenage by 11:45 .
i 'm looking for a train leaving from norwich .
i 'm looking to go to cambridge on tuesday
i 'll be departing from norwich on wednesday .
i 'd like to leave after 13:45 .
i 'm looking for a train departing from cambridge .
i 'm looking for a train on sunday from cambridge .
can you verify that it 'll arrive in stevenage by 09:00 ?
i 'll be going to london liverpool street on sunday .
i 'd prefer to leave after 13:45 please
i 'd like to depart on sunday .
i 'd like to travel on sunday .
i 'm looking for a train that leaves on thursday and arrives by 09:15
i 'm leaving from cambridge where i 'm now .
i 'm looking for a train going to birmingham new street and should leave on thursday
i 'm departing from cambridge and want the train to arrive by 17:15 .
i 'd like to leave cambridge and go to london liver pool .
i 'd like to go this saturday to cambridge .
it 'll be for thursday .
i 'm going to cambridge from broxbourne .
yes that 'd be fine please book for 6 people .
i 'm looking for a train from norwich on monday .
i 'd like to leave cambridge after 09:15 .
i 'd like to book a train for monday departing from kings lynn and arriving at cambridge at 14:45 .
i 'm looking for a train to london kings cross that departs after 08:15 .
i 'd like to leave on thursday
i 'm going to kings lynn
i 'd like to leave on sunday .
i 'd like to leave on tuesday .
i 'm looking for a train .
i 'm looking to go to london kings cross and depart from cambridge .
i 'm trying to find a train leaving after 15:15 to go to broxbourne .
i 'm looking at traveling on sunday .
i 'll be leaving from cambridge on friday .
i 'm heading to cambridge .
i 'll need to arrive by 17:15 .
i 'll be leaving on wednesday .
i 'll be leaving saturday morning at 08:45
i 'd like to arrive in cambridge by 14:15 .
i 'm looking to book a ticket for 2 people for the tuesday train leaving from cambridge and arriving by 10:30 at broxbourne
i 'll be traveling on monday and need to arrive by 14:15
i 'd like to travel on monday after 12:00 .
i 'd be leaving from cambridge .
i 'm looking for train going to cambridge please .
i 'll be leaving from brookshite .
i need to leave after 17:45 so anytime after that 'd be great .
i 'm looking for a train from cambridge to leicester .
i 'll be leaving on monday .
i 'd like to leave after 09:00 on sunday .
i 'm going to cambridge .
i 'm leaving on sunday and need to arrive by 13:00
i 'd like to leave on saturday from london liverpool street
i 'm looking for a train schedule that departs from cambridge and should go to ely .
i 'd like to travel this wednesday and need to leave after 08:15 .
i 'd like to go on wednesday .
i 'd like to leave from stevenage on thursday and get to cambridge by 9:15 .
i 'm leaving leicester and i need to leave by 20:30 .
i 'm looking for a train leaving after 11:45
that 'd work for me can i get a train for 7 people ?
i 'd like to go to birmingham new street and the train should leave after 14:15 .
i 'm looking for a train that goes to cambridge around 10:45
i 'm looking for one that will arrive by 15:00
i 'd like to leave after 18:15 please .
i 'll be departing from norwich .
i 'd like to leave after 14:30 .
i 'm looking for a train departing from london liverpool street on thursday to go to cambridge .
i 'm looking for a train from kings lynn .
i 'd like to travel on thursday .
i 'm looking for a train .
i 'm looking for a train to peterborough .
i 'm looking to go from cambridge to peterborough and leave after 14:15
i 'd like to leave on tuesday and need to arrive by 09:00 .
i 'm looking for a train departing from london kings cross .
i 'd like to arrive in cambridge on tuesday by 17:00 .
yes i 'd like to book it for 8 people and i 'll need the reference number .
i 'd like to leave after 12:15 .
i 'll depart any time so long as the train arrives by 20:45 .
i 'll be departing from cambridge .
i 'll be going to stansted airport .
i 'll be travelling on monday .
i 'm looking for a train that departs cambridge after 21:00 on thursday .
i 'm looking for a train to cambridge on saturday .
i 'd like to travel on saturday .
i 'd like to leave after 11:15 .
yes i 'd like to go from leicester to cambridge and leave after 11:15 please .
i 'm going to cambridge and need to leave after 12:00
i 'm looking for a train from london kings cross and leaves on wednesday
i 'd like to leave on tuesday and arrive by 13:30 .
i 'm looking for a train on friday that leaves after 21:45 .
i 'll be departing from london kings cross after 19:00 .
i 'm looking for a train leaving london kings cross and arriving by 14:30 .
i 'll need to leave from cambridge .
i 'd like to travel on a thursday .
i 'd like to catch a train leaving cambridge on sunday .
i 'm looking for a train that leaves bishops stortford and goes to cambridge
i 'd like to arrive in cambridge by 8:15 .