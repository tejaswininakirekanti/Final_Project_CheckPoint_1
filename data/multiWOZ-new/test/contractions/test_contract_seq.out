O O O O O O B-depart I-depart O B-leave O
O O O O O O O B-leave O
O O O O O O O O B-dest O
O O O O O B-depart O B-day O
O O O O O O O B-people O
O O O O O O O O B-depart I-depart I-depart
O O O O B-day O O O O O B-leave O
O O O O B-depart I-depart I-depart O O O B-dest O
O O O O O O B-day O
O O O O O O O B-depart O B-day O
O O O O O O O O O B-dest I-dest I-dest O O O B-leave O
O O O O O O O O O B-depart O B-day O
O O O O O B-depart O
O O O O B-day O
O O O O O O O O O O B-depart O B-day O
O O O O O O O O O O O O B-dest O B-arrive O
O O O O O O O O B-depart O
O O O O O O B-dest O B-day
O O O O O B-depart O B-day O
O O O O O O B-leave O
O O O O O O O O B-depart O
O O O O O O O B-day O B-depart O
O O O O O O O O B-dest O O O
O O O O O B-dest I-dest I-dest O O O
O O O O O B-leave I-leave O
O O O O O O B-day O
O O O O O O B-day O
O O O O O O O O O B-day O O O B-arrive
O O O O B-depart O O O O O
O O O O O O O O B-dest I-dest I-dest O O O O B-day
O O O O B-depart O O O O O O O B-arrive O
O O O O O B-depart O O O O O O O
O O O O O O B-day O B-dest O
O O O O B-day O
O O O O B-dest O B-depart O
O O O O O O O O B-people O O
O O O O O O O B-depart O B-day O
O O O O O B-depart O B-leave O
O O O O O O O O B-day O O B-depart I-depart O O O B-dest O B-arrive O
O O O O O O O B-dest I-dest I-dest O O O B-leave O
O O O O O O B-day
O O O O B-dest I-dest
O O O O O O B-day O
O O O O O O B-day O
O O O O O B-depart O
O O O O O O B-dest I-dest I-dest O O O B-depart O
O O O O O O O O O B-leave O O O B-dest O
O O O O O O B-day O
O O O O O B-depart O B-day O
O O O O B-dest O
O O O O O O B-arrive O
O O O O O B-day O
O O O O B-day O O B-leave
O O O O O O O O B-arrive O
O O O O O O O O B-people O O O B-day O O O B-depart O O O B-arrive O B-dest
O O O O O B-day O O O O O B-arrive
O O O O O O B-day O B-leave O
O O O O O B-depart O
O O O O O O O B-dest O O
O O O O O B-depart O
O O O O O B-leave O O O O O O O O
O O O O O O O B-depart O B-dest O
O O O O O B-day O
O O O O O B-leave I-leave O B-day O
O O O O B-dest O
O O O O B-day O O O O O B-arrive
O O O O O O B-day O O O O
O O O O O O O O O O B-depart O O O O B-dest O
O O O O O O B-day O O O O O B-leave O
O O O O O O B-day O
O O O O O O B-depart O B-day O O O O O O O
O O O B-depart O O O O O O B-leave O
O O O O O O O O B-leave
O O O O O O O O O O O B-people O O
O O O O O O B-dest I-dest I-dest O O O O O O B-leave O
O O O O O O O O O B-dest O B-arrive
O O O O O O O O O B-arrive
O O O O O O B-leave O O
O O O O O B-depart O
O O O O O O B-leave O
O O O O O O O O B-depart I-depart I-depart O B-day O O O B-dest O
O O O O O O O B-depart I-depart O
O O O O O O B-day O
O O O O O B-depart O
O O O O O O O B-dest O
O O O O O O B-depart O O O O O B-leave
O O O O O O B-day O O O O O B-arrive O
O O O O O O O O B-depart I-depart I-depart O
O O O O O O B-dest O B-day O B-arrive O
O O O O O O O O B-people O O O O O O O O O
O O O O O O B-leave O
O O O B-leave O O O O O O O O B-arrive O
O O O O O B-depart O
O O O O O B-dest I-dest O
O O O O O B-day O
O O O O O O O O B-depart O B-leave O B-day O
O O O O O O O B-dest O B-day O
O O O O O O B-day O
O O O O O O B-leave O
O O O O O O O O O B-dest O O O O O O
O O O O B-dest O O O O O B-leave
O O O O O O O B-depart I-depart I-depart O O O B-day
O O O O O O B-day O O O B-arrive O
O O O O O O O B-day O O O B-leave O
O O O O O B-depart I-depart I-depart O B-leave O
O O O O O O O B-depart I-depart I-depart O O O B-arrive O
O O O O O O B-depart O
O O O O O O O B-day O
O O O O O O O O B-depart O B-day O
O O O O O O O O B-depart I-depart O O O B-dest
O O O O O O O O B-arrive O